src/EFCore.PG/ValueGeneration/Internal/NpgsqlUUID7ValueGenerator.cs

```diff
@@ -0,0 +1,108 @@
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace Npgsql.EntityFrameworkCore.PostgreSQL.ValueGeneration.Internal;
+
+/// <summary>
```

Comment:
**Chore:** Let's replace this summary, and the two below it, by the "pubternal" summary (copied from similar types elsewhere in the project):

```cs
/// <summary>
///     This API supports the Entity Framework Core infrastructure and is not intended to be used
///     directly from your code. This API may change or be removed in future releases.
/// </summary>
```

Reply:
Everything else in that ValueGeneration folder looks like it uses that wording so maybe keep it like that to not be different to other value generators

Reply:
It's true that the EF standard is to have the standard "pubternal" summary on everything - both the class and anything public/protected on it. This probably isn't applied consistently in the provider at the moment, and also isn't extremely important.

Reply:
Updated summary description